src/core/gosling-track-model.ts

```diff
@@ -162,13 +162,8 @@ export class GoslingTrackModel {
         // zero baseline
         SUPPORTED_CHANNELS.forEach(channelKey => {
             const channel = spec[channelKey];
-            if (
-                IsChannelDeep(channel) &&
-                typeof channel.zeroBaseline === 'undefined' &&
-                channel.type !== 'genomic' &&
-                ['x', 'y'].includes(channelKey)
```

Comment:
I think using `zeroBaseline` by default for all quantitative channels (e.g., `size`, `color`, `y`) makes more sense. So, updated this part accordingly.